CHANGELOG.md

```diff
@@ -3,6 +3,7 @@ All notable changes to this project will be documented in this file.
 
 ## [Unreleased]
 ### Added
+- OxyPlot.Series.HeatMapSeries.GetValueForLabels (#256)
```

Comment:
The previous should have been: GetValueForLabels should be renamed to GetValueForLabel.

This will be in the next sync.